pkg/security/probe/model.go

```diff
@@ -966,8 +1017,9 @@ type Event struct {
 	Umount           UmountEvent           `field:"-"`
 	InvalidateDentry InvalidateDentryEvent `field:"-"`
 
-	resolvers         *Resolvers         `field:"-"`
-	processCacheEntry *ProcessCacheEntry `field:"-"`
+	resolvers           *Resolvers         `field:"-"`
```

Comment:
nit: you can remove the tag here, see my previous comment

Reply:
👍🏻

Reply:
I get a invalid memory address or nil pointer dereference error when I run go generate without it :(